### PR TITLE
Updating the Persian translation message file.

### PR DESCRIPTION
**Pull request by NabiKAZ**
Updating the Persian translation message file.

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: framework/messages/fa/yii.php [hunk 1]
```diff
@@ -45,8 +45,8 @@
     'Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}.' => 'نمایش <b>{begin, number} تا {end, number}</b> مورد از کل <b>{totalCount, number}</b> مورد.',
     'The combination {values} of {attributes} has already been taken.' => 'مقدار {values} از {attributes} قبلاً گرفته شده است.',
     'The file "{file}" is not an image.' => 'فایل "{file}" یک تصویر نیست.',
-    'The file "{file}" is too big. Its size cannot exceed {formattedLimit}.' => 'حجم فایل "{file}" بسیار بیشتر می باشد. حجم آن نمی تواند از {formattedLimit} بیشتر باشد.',
-    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => 'حجم فایل "{file}" بسیار کم می باشد. حجم آن نمی تواند از {formattedLimit} کمتر باشد.',
```
Comment (salehhashemi1992) [inline framework/messages/fa/yii.php:49]: Why did you remove that space between آن and نمی ? this is not true

Comment (NabiKAZ) [reply framework/messages/fa/yii.php:49]: You're right. It seems to be a problem with my editor. I'll double-check the spaces (نیم‌فاصله‌ها). If I do this now, there is a possibility of conflict with #19996. Thankful.
